src/preact/base-element.js

```diff
@@ -726,6 +726,19 @@ function collectProps(Ctor, element, ref, defaultProps) {
       devAssert(def.attrs);
       value = def.parseAttrs(element);
     }
+    if (def.attrPrefix) {
+      const currObj = {};
```

Comment:
Another critical thing: we need modify the `shouldMutationBeRerendered` to test for the prefix. Otherwise changes to the relevant attributes will be missed.